### PR TITLE
Update cobra help message to be aligned with out official docs

### PR DESCRIPTION
**Pull request by pchico83**
While working on our CLI 3.0 official docs, I realized we are using different terminology than in our cobra help messages.

This PR standardized to use the same terminology than in our docs, and unify the text messages too

### CODE REVIEW

File: cmd/up/up.go [hunk 1]
```diff
@@ -374,17 +374,17 @@ okteto up my-svc -- echo this is a test
...
 	}
 
-	cmd.Flags().StringVarP(&upOptions.ManifestPath, "file", "f", "", "path to the Okteto manifest file")
-	cmd.Flags().StringVarP(&upOptions.Namespace, "namespace", "n", "", "namespace where the up command is executed")
-	cmd.Flags().StringVarP(&upOptions.K8sContext, "context", "c", "", "context where the up command is executed")
-	cmd.Flags().StringArrayVarP(&upOptions.Envs, "env", "e", []string{}, "envs to add to the development container")
-	cmd.Flags().IntVarP(&upOptions.Remote, "remote", "r", 0, "configures remote execution on the specified port")
-	cmd.Flags().BoolVarP(&upOptions.Deploy, "deploy", "d", false, "Force execution of the commands in the 'deploy' section of the okteto manifest (defaults to 'false')")
-	cmd.Flags().BoolVarP(&upOptions.ForcePull, "pull", "", false, "force dev image pull")
+	cmd.Flags().StringVarP(&upOptions.ManifestPath, "file", "f", "", "thepath to the Okteto Manifest")
```
Comment (andreafalzetti) [inline cmd/up/up.go:377]:
```suggestion
	cmd.Flags().StringVarP(&upOptions.ManifestPath, "file", "f", "", "the path to the Okteto Manifest")
```

File: cmd/context/use.go [hunk 2]
```diff
@@ -84,8 +77,8 @@ Or a Kubernetes context:
 		},
 	}
 
-	cmd.Flags().StringVarP(&ctxOptions.Token, "token", "t", "", "API token for authentication")
-	cmd.Flags().StringVarP(&ctxOptions.Namespace, "namespace", "n", "", "namespace of your okteto context")
+	cmd.Flags().StringVarP(&ctxOptions.Token, "token", "t", "", "API token for authentication. Use this when scripting or if you don't want to use browser-based authentication")
+	cmd.Flags().StringVarP(&ctxOptions.Namespace, "namespace", "n", "", "overwrite the current Okteto Namespacet")
```
Comment (andreafalzetti) [inline cmd/context/use.go:81]:
```suggestion
	cmd.Flags().StringVarP(&ctxOptions.Namespace, "namespace", "n", "", "overwrite the current Okteto Namespace")
```

File: cmd/pipeline/list.go [hunk 3]
```diff
@@ -59,16 +59,16 @@ func list(ctx context.Context) *cobra.Command {
...
 		RunE: func(cmd *cobra.Command, args []string) error {
 			return pipelineListCommandHandler(ctx, flags, contextCMD.NewContextCommand().Run)
 		},
 	}
 
-	cmd.Flags().StringVarP(&flags.context, "context", "c", "", "context where the pipelines are deployed (defaults to the current context)")
-	cmd.Flags().StringArrayVarP(&flags.labels, "label", "", []string{}, "tag and organize dev environments using labels (multiple --label flags accepted)")
-	cmd.Flags().StringVarP(&flags.namespace, "namespace", "n", "", "namespace where the pipelines are deployed (defaults to the current namespace)")
+	cmd.Flags().StringVarP(&flags.context, "context", "c", "", "overwrite the current Okteto Context")
+	cmd.Flags().StringArrayVarP(&flags.labels, "label", "", []string{}, "tag and organize Development Environments using labels (multiple --label flags accepted)	")
```
Comment (andreafalzetti) [inline cmd/pipeline/list.go:70]:
```suggestion
	cmd.Flags().StringArrayVarP(&flags.labels, "label", "", []string{}, "tag and organize Development Environments using labels (multiple --label flags accepted)")
```

File: cmd/preview/list.go [hunk 4]
```diff
@@ -65,7 +65,7 @@ func List(ctx context.Context) *cobra.Command {
 	flags := &listFlags{}
 	cmd := &cobra.Command{
 		Use:   "list",
-		Short: "List all preview environments",
+		Short: "List all your Preview Environments",
```
Comment (andreafalzetti) [inline cmd/preview/list.go:68]: nit: it also lists global-scoped preview enviroments

**Review by andreafalzetti — APPROVED** [review]

File: pkg/syncthing/completion.go [hunk 5]
```diff
@@ -169,7 +169,7 @@ func (wfc *waitForCompletion) isCompleted() bool {
 		}
 	}
 	if !wfc.sy.IsAllOverwritten() {
-		oktetoLog.Info("synced completed, but overwrites not sent, retrying...")
+		oktetoLog.Info("synced completed, but overwrite not sent, retrying...")
```
[the remark this answers is not in the document]
Comment (pchico83) [reply pkg/syncthing/completion.go]: nope, undone :-)

**Comment by pchico83** [comment]
Thanks @andreafalzetti for the detailed review!

**Pull request merged**